# Device ID and capabilities
string id
string type
string[] traits

# Device names
string name
string[] nicknames
string[] default_names

# Status reporting info
bool will_report_state
string room_hint

# Make and manufacture
string manufacturer
string model
string sw_version
string hw_version

# Alternate, local device ID
string device_id

